forc/GFS/README.md

```diff
@@ -0,0 +1,94 @@
+## Description
+
+* This package generates DATM input data for the forcing fluxes at the sea surface from the NCEP operational GFSv16 outputs.
+
+## Inputs of conversion in conv_gfs2datm_long_beta4.f
+
+* INPUT Sources : from the output of GFSv16,
+
+I. ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.sfluxgrbf000.grib2 :: Input data are from the GFSv16 outs of initial analysis fields, 
+  except following missing variables in the initial file.
+
+II. ./gdas.${datb}/${hhb}/atmos/gdas.t${hhb}z.sfluxgrb${hprep}.grib2 :: 6 hour forecast fields at the analysis time
+      ==> PRATE_surface, UFLX_surface, VFLX_surface
+
+III. ./gdas.${dat}/${hh}/atmos/gdas.t${hh}z.atmf000.nc  ==> delz
+
+| Input variable |	Source	| Output variable|
+| --- | --- | ---|
+| longitude	| I| lon |
+| latitude |I |lat |
+| time | I| time |
+|DLWRF_surface  |I |DLWRF |
+|ULWRF_surface  | I|ULWRF |
+| DSWRF_surface |I |DSWRF |
+|DSWRF_surface |I |vbdsf_ave |
+|DSWRF_surface |I | vddsf_ave|
+|DSWRF_surface |I |nbdsf_ave |
+|DSWRF_surface |I | nddsf_ave|
+| UFLX_surface|II | dusfc|
+|VFLX_surface |II |dvsfc |
+|SHTFL_surface |I | shtfl_ave|
+|LHTFL_surface |I |lhtfl_ave |
+| PRATE_surface|I |totprcp_ave |
+| UGRD_10maboveground|I |u10m |
+|VGRD_10maboveground |I |v10m |
+|delz |III |hgt_hyblev1 |
+|PRES_surface |I | psurf|
+|PRATE_surface |II |precp |
+|PRATE_surface |II | fprecp|
```

Comment:
Are these the same variable?

Reply:
Yes, precp and fprecp are estimated from PRAYE_surface

Reply:
Ok. I see `fprecp` (frozen precipitation) is derived empirically.